pnpm-lock.yaml

```diff
@@ -1,9 +1,5 @@
 lockfileVersion: '6.0'
 
-settings:
-  autoInstallPeers: true
-  excludeLinksFromLockfile: false
-
```

Comment:
You _might_ be on an older version of pnpm. Don't think this should've been removed.

Reply:
Oh, did it just get moved to the bottom of the file? That's weird.

Reply:
haha not sure, we think it's in the original place now tho